x/staking/types/msg.go

```diff
@@ -131,6 +134,15 @@ func (msg MsgCreateValidator) ValidateBasic() error {
 		)
 	}
 
+	msd, _ := sdk.NewIntFromString(MinSelfDelegation)
```

Comment:
Is it worth checking that this succeeded?

Reply:
`MinSelfDelegation` is const value so I think it will be always ok. But to ensure  no unpredictable situation I've added new validation at here: [Ensure MinSelfDelegation value can convert to int](https://github.com/CudoVentures/cosmos-sdk/pull/15/commits/990597ad073a60cfb2b5c49c55ffe5d2d30b6cab)